app/experimenter/experiments/constants/__init__.py

```diff
@@ -0,0 +1,2 @@
+from experimenter.experiments.constants.legacy import ExperimentConstants  # noqa: F401
+from experimenter.experiments.constants.nimbus import NimbusConstants  # noqa: F401
```

Comment:
Didn't want to use a `__all__`?

Reply:
Oh I discovered that `__all__` is only used when you `import *`, not when you import individually by name, and since we don't use `import *` anywhere it doesn't seem to provide any value?

Reply:
I meant so that the `noqa` wouldn't be needed.

Reply:
Yeah I know but the other thing with `__all__` is it's lazily evaluated so we had a situation last week where we made changes to the module but forgot to make them in `__all__` and it passed through all the checks and made it into `main` and wasn't noticed until it broke when I was poking around at stuff, so it seems to

- not provide value
- introduce possible risk/complexity

so just disabling that one lint check seems less risky? What do you think?

Reply:
I'm good with disabling the check for these.